recipe/builder.py

```diff
@@ -14,13 +14,29 @@ def install():
     """ populate the environment variable added by the patch and actually
         install with pip
     """
+
+    cuda_version = os.environ.get('cuda_compiler_version', None)
+
+    try:
+        float(cuda_version)
+        is_cuda = True
+    except (TypeError, ValueError):
+        is_cuda = False
+
     if WIN:
         cmake_args = (
+            "-G \"Ninja\"\n"
```

Comment:
The quotes seem to confuse CMake and we don't need them. So let's drop them

```suggestion
            "-G Ninja\n"
```